### PR TITLE
231 service worker

### PR DESCRIPTION
**Pull request by GithinjiHans**

### CODE REVIEW

**Comment by Tushar-4781** [comment]
I can feel the pain @GithinjiHans ;D
This will be the first PR of my life where I see 45 commits. Cheers... lol ....

File: src-sw.js [hunk 1]
```diff
@@ -2,4 +2,8 @@ importScripts('https://storage.googleapis.com/workbox-cdn/releases/3.0.0/workbox
 
 console.log('service worker customized');
 
+workbox.routing.registerRoute(
+    new RegExp('https://zinzen-qqu97eop7-tijlleenders.vercel.app/Home/MyGoals),
```
Comment (Tushar-4781) [inline src-sw.js:6]: I think you are missing the quote ( ' ). the URL is not in quotes

**Comment by tijlleenders** [comment]
Maybe use a [vite plugin](https://vite-plugin-pwa.netlify.app/) to make this work if the [public directory](https://vitejs.dev/guide/assets.html#the-public-directory) doesn't fix it?

There's also a [working example PWA](https://github.com/imranhsayed/react-pwa-workbox-example) to compare with.

**Comment by tijlleenders** [comment]
Can #387 be closed and deleted in favor of this one?

**Pull request closed without merging**